### DISCUSSION TITLE
Behavior of the "any" quality profile

### DISCUSSION THREAD

**Discussion by cmess77** [lardbit/nefarious#307] [Q&A] [answered]
Thanks for this amazing project, I love it! I just have a quick question about the behavior of the any quality profile. I tried digging around in the source, but couldn't really find an answer, likely due to my lack of experience with TypeScript. 

When the any profile is selected, does the application take the first quality it finds? Or, does it slowly lower the quality as long as it can't find a torrent?

**Comment by lardbit** [comment, marked as answer]
The matching logic will always choose the torrent with the most seeders
that's within the quality profile chosen.  So, "any" would simply choose
the torrent with the most seeders and not care about the quality.

You can play with the quality profiles by specifying min/max sizes on top
of the actually qualities (e.g. 720p).  See the settings page to change the
quality profiles.
